base/exports.jl

```diff
@@ -993,6 +993,7 @@ export
     error,
     rethrow,
     retry,
+    default_retry_check,
```

Comment:
don't think we should be exporting `default_worker_pool` either, at least not under that name